dotnet/docs/TELEMETRY.md

```diff
@@ -36,7 +36,13 @@ Log levels used in SK:
 Enable logging for Kernel instance:
 
 ```csharp
-var kernel = new KernelBuilder().WithLoggerFactory(loggerFactory);
+IKernelBuilder builder = Kernel.CreateBuilder();
+
+// Assuming loggerFactory is already defined.
+builder.Services.AddSingleton(loggerFactory);
```

Comment:
nit: We can also add simpler syntax as an example:
```suggestion
builder.Services.AddSingleton(loggerFactory); // or builder.Services.AddLogging();
```

Reply:
But this wouldn't take a custom ILoggerFactory no?

Reply:
Correct, it will create default one, but it's still possible to configure it using logging builder:
```csharp
builder.Services.AddLogging(loggingBuilder => loggingBuilder.AddOpenTelemetry());
```

And here is custom ILoggerFactory version:
```csharp
builder.Services.AddSingleton(LoggerFactory.Create(loggingBuilder => loggingBuilder.AddOpenTelemetry()));
```